jenkins/databricks/params.py

```diff
@@ -23,24 +23,29 @@
 source_tgz = 'spark-rapids-ci.tgz'
 tgz_dest = '/home/ubuntu/spark-rapids-ci.tgz'
 base_spark_pom_version = '3.0.1'
+# this is odd but with Databricks 8.2 it reports Spark version 3.0.1
+# but its really 3.1.1, so base spark pom will be 3.1.1 but we want to
+# install them as 3.1.0-databricks. Other Databricks versions
+# this is the same as base_spark_pom_version.
+base_spark_version_to_install_databricks_jars = '3.0.1'
```

Comment:
Should default to `base_spark_pom_version` rather than `3.0.1`?  That way when `base_spark_pom_version` updates so does this automatically unless overridden on the cmdline.

Reply:
yes